xarray/core/common.py

```diff
@@ -572,12 +598,53 @@ def resample(self, freq=None, dim=None, how=None, skipna=None,
             object will be returned without attributes.
         **indexer : {dim: freq}
             Dictionary with a key indicating the dimension name to resample
-            over and a value corresponding to the resampling frequency.
+            over and a value corresponding to the resampling frequency. e.g.
+            time="M"
 
         Returns
         -------
         resampled : same type as caller
             This object resampled.
+            
+        Examples
+        --------
+        Resample monthly time-series data to seasonal data:
+        
+        >>> da = xr.DataArray(np.linspace(0, 11, num=12),
+        ...                   coords=[pd.date_range('15/12/1999',
+        ...                           periods=12, freq=pd.DateOffset(months=1))],
+        ...                   dims='time')
+        >>> da
+        <xarray.DataArray (time: 12)>
+        array([  0.,   1.,   2.,   3.,   4.,   5.,   6.,   7., 8.,   9.,  10.,  11.])
+        Coordinates:
+          * time     (time) datetime64[ns] 1999-12-15 2000-01-15 2000-02-15 ...
+        >>> da.resample(time="Q-DEC").mean()
+        <xarray.DataArray (time: 4)>
+        array([ 1.,  4.,  7., 10.])
+        Coordinates:
+          * time     (time) datetime64[ns] 2000-02-29 2000-05-31 2000-08-31 2000-11-30
```

Comment:
Maybe this would be a good place to show upsampling, e.g., from monthly to daily? Maybe:
```
In [10]: da.resample(time='1D').interpolate('linear')
Out[10]:
<xarray.DataArray (time: 337)>
array([ 0.      ,  0.032258,  0.064516, ..., 10.935484, 10.967742, 11.      ])
Coordinates:
  * time     (time) datetime64[ns] 1999-12-15 1999-12-16 1999-12-17 ...
```

Reply:
Done